composer.json

```
@@ -24,7 +24,7 @@
   "require": {
     "php": "^8.0",
     "ext-mbstring": "*",
-    "psr/http-message": "^1.0",
+    "psr/http-message": "^1.0 || ^2.0",
```

@PaolaRuby I haven't added those type declarations earlier because you're not allowed to add them if the interface does ot specify them as well.

Therefore we can't be compatible to v1 & v2 at the same time.

```suggestion
    "psr/http-message": "^2.0",
```

---

i didn't know, sorry